modules/minileven/theme/pub/minileven/functions.php

```diff
@@ -237,5 +245,15 @@ function minileven_get_gallery_images() {
 function minileven_show_featured_images() {
 	$enabled = ( is_home() || is_search() || is_archive() ) ? true : false;
 
+	/**
+	 * Filter where featured images are displayed in the Mobile Theme.
+	 * 
+	 * For example, by defining $enabled as above, featured images will display 
```

Comment:
What about this:

```
/**
 * Filter where featured images are displayed in the Mobile Theme.
 * 
 * By setting $enabled to true or false using functions like is_home() or 
 * is_archive(), you can control where featured images are be displayed.
 *
 * @since 3.2.0
 *
 * @param bool $enabled True if featured images should be displayed, false if not.
 */
```